build-scripts/builder/ubuntu/target/ubuntu

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+arg1=$1
+
+case $arg1 in
+    "setup")
+        echo "setup"
+        curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr --output utr
+        chmod +x ./utr
+        unity-downloader-cli -u 2021.2 -c Editor --fast --wait
+        ;;
+    "build")
+        echo "build"
+        export UTR_VERSION=0.12.0
+        ./utr --suite=playmode --platform=StandaloneLinux64  --editor-location=.Editor --testproject=testproject --player-save-path=build/players --artifacts_path=build/logs --scripting-backend=mono --build-only --testfilter=Unity.Netcode.RuntimeTests --extra-editor-arg=-batchmode --extra-editor-arg=-nographics
```

Comment:
You should be able to use `--extra-editor-arg=-batchmode --extra-editor-arg=-nographics` on all platforms and get a perf boost.

Reply:
added the extra editor args